### ISSUE TITLE
[Feature request]Credentials are in the open

### ISSUE THREAD

**Issue by T00ManySecrets** [iEvgeny/cctv-viewer#74]
Could we possibly have a way to lock the sidebar with a password or something. For cameras with credentials anyone can see username and password. 
Love you app. Use it a lot.

**Comment by iEvgeny** [comment]
Of course! Use the -k command line key when launching the application.
More information `$ cctv-viewer --help-all`

**Comment by T00ManySecrets** [comment]
Nice!! Thank you
